### PR TITLE
Rendering issue

### PR DESCRIPTION
**Pull request by alxschwrz**
Added rendering option for all examples and environments as described in issue #3

### CODE REVIEW

File: examples/ground_robot.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def main():
         for i in range(n_steps):
             # action = env.action_space.sample()
             action = defaultAction
-            env.render()
+            #env.render()
```
Comment (maxspahn) [inline examples/ground_robot.py:19]: You can actually remove that line entirely.

File: examples/ground_robot_arm.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def main():
         print("Starting episode")
         for i in range(n_steps):
             action = defaultAction
-            env.render()
+            #env.render()
```
Comment (maxspahn) [inline examples/ground_robot_arm.py:17]: Remove

**Review by maxspahn — CHANGES_REQUESTED** [review]
Nice work. Could you remove the commented lines with env.render(). So that it is clean.

**Comment by alxschwrz** [comment]
Thanks. I deleted the deprecated render calls in ground_robot.py and ground_robot_arm.py in commit fc23dfbff18e4c93401653384a93421c133a1b10.

**Review by maxspahn — APPROVED** [review]

**Pull request merged**